build_linux.sh

```diff
@@ -9,7 +9,7 @@ export CODEC2DIR=$FREEDVGUIDIR/codec2
 export LPCNETDIR=$FREEDVGUIDIR/LPCNet
 
 # change this when working on combined codec2/freedv-gui changes
-CODEC2_BRANCH=master
+CODEC2_BRANCH=ms-reliable-text
 
```

Comment:
@tmiw should we change this back to `master` prior to merge?

Reply:
Done.